cmd/gitbase/command/server.go

```diff
@@ -66,12 +79,54 @@ func (c *Server) Execute(args []string) error {
 		{Password: c.Password},
 	}
 
+	var tracer opentracing.Tracer
+	if c.TraceEnabled {
+		if os.Getenv("JAEGER_SERVICE_NAME") == "" {
+			os.Setenv("JAEGER_SERVICE_NAME", TracerServiceName)
+		}
+
+		cfg, err := config.FromEnv()
+		if err != nil {
+			logrus.WithField("error", err).
+				Fatal("unable to read jaeger environment")
+			return err
+		}
+
+		logger := &jaegerLogrus{logrus.WithField("subsystem", "jaeger")}
+
+		t, closer, err := cfg.NewTracer(
+			config.Logger(logger),
+		)
+
+		if err != nil {
+			logrus.WithField("error", err).Fatal("unable to initialize tracer")
+			return err
+		}
+
+		tracer = t
+		defer closer.Close()
+
+		jaegerHost := os.Getenv("JAEGER_AGENT_HOST")
```

Comment:
the same as the previous comment.

Reply:
I was not able to get those parameters from config. These are set in UDP transport and they don't have any public way to get them.

https://github.com/jaegertracing/jaeger-client-go/blob/master/transport_udp.go#L49-L75

Reply:
Deleted generation of the endpoint in logs as it's not really needed.